_docs_v7/Physical-Definition.md

```diff
@@ -158,3 +158,27 @@ The following modifications are allowed:
   - Curvature corrections are currently not implemented.
 
 Modifications from each of these three groups can be combined, for example `SST_OPTIONS= V2003m, VORTICITY, SUSTAINING`
+
+## Transition Models ##
+
+| Solver | Version |
+| --- | --- |
+| `*_RANS` | 7.4.0 |
+
+This section describes how to setup transition models for RANS simulations. Transition is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`.
+A transition model can then be selected via the option `KIND_TRANS_MODEL`.
+Different submodels and correlations are specified via the different options listed below.
+
+The following modifications are allowed:
+- Versions:
+  - `LM2015` - Correction to include stationary crossflow instabilities. It has to be used only in 3D problems. The RMS of roughness used in this model has to be set through the separate option `HROUGHNESS`.
+- Correlations:
+  - `MALAN` - This is the default correlation when the LM model is coupled with the `SA` turbulence model.
+  - `SULUKSNA` - This should be used only if the `SST` model is used. It requires a formulation of
```

Comment:
incomplete sentence

Reply:
I also added some figures for the validation of the E387 test case. Please, check if those are good, and then I'll merge this, thank you.